File: quicksort/exercises/exercise_4_2.js
// Write a recursive function to count the number of items in a list
function recursionCount(arr) {
  let total = 0;
  for(let i = 0; i < arr.length; i++){
    total += 1;
  }
  return total;
}

console.log(recursionCount([2,4,6]));
console.log(recursionCount([1,2,3,4,5]));

File: quicksort/exercises/exercise_4_2.rb
#Write a recursive function to count the number of items in a list
def recursion_count(arr)
  total = 0
  arr.each do |num|
    total += 1
  end
  return total
end

puts recursion_count([2,4,6])

puts recursion_count([1,2,3,4,5])

